packages/jest-config/src/index.js

```diff
@@ -17,24 +22,34 @@ import readConfigFileAndSetRootDir from './read_config_file_and_set_root_dir';
 
 function readConfig(
   argv: Argv,
-  packageRoot: string,
+  packageRootOrConfig: Path | InitialOptions,
   // Whether it needs to look into `--config` arg passed to CLI.
   // It only used to read initial config. If the initial config contains
   // `project` property, we don't want to read `--config` value and rather
   // read individual configs for every project.
   skipArgvConfigOption?: boolean,
+  parentConfigPath?: ?Path,
 ): {
   configPath: ?Path,
   globalConfig: GlobalConfig,
   hasDeprecationWarnings: boolean,
   projectConfig: ProjectConfig,
 } {
   let rawOptions;
-  let configPath;
+  let configPath = null;
```

Comment:
Is there anywhere that _requires_ a `configPath` to be set other than `readConfigFileAndSetRootDir`?